codegenerator/cli/templates/static/codegen/src/EventModLookup.res

```diff
@@ -0,0 +1,109 @@
+open Belt
+type eventMod = module(Types.InternalEvent)
+
+type errorKind = WildcardSighashCollision | Duplicate
+type eventError = {eventMod: eventMod, errorKind: errorKind}
+module ContractEventMods = {
+  type t = {
+    all: array<eventMod>,
+    byContractName: dict<eventMod>,
+  }
+
```

Comment:
Strange this wasn't tracked as a move... but there are quite a lot of changes here as well.